packages/manager/modules/billing/src/autoRenew/ssh/user-ssh.service.js

```diff
@@ -92,6 +94,19 @@ export default /* @ngInject */ function UserAccountSshService(
     return results.flat();
   };
 
+  self.getCloudProjects = function getCloudProjects() {
+    return iceberg('/cloud/project')
+      .query()
+      .expand('CachedObjectList-Pages')
+      .execute()
+      .$promise.then(({ data }) =>
+        map(filter(data, { status: 'ok' }), (project) => ({
+          id: project.project_id,
+          description: project.description,
+        })),
```

Comment:
Please rewrite this part to avoid using lodash:
```suggestion
        data.filter((project) => project.status === 'ok')
          .map((project) => ({
            id: project.project_id,
            description: project.description,
          })),
```

Also, you can now use this function in `getCloudSshList`